advisor-ui/src/components/AddConstraint/AddConstraint.jsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import "./AddConstraint.css";
+import { Button } from "@mui/material";
 
 export default function AddConstraint({ openPopover }) {
   return (
-    <div className="addconstraint" onClick={openPopover}>
-      <p>Add Constraint</p>
-    </div>
+    <Button variant="outlined" className="addconstraint" onClick={openPopover}>
+      ADD CONSTRAINT
```

Comment:
You could use CSS to control this, which would make the text consistent with other text in your project.

`text-transform: uppercase`

Just FYI, there's definitely convincing arguments to be made for handling it both ways.